Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -19,24 +19,52 @@ struct GenerateCommand: ParsableCommand, HasTrackableParameters {
     )
     var path: String?
 
+    @Argument(help: """
+    A list of targets in which you'd like to focus. \
+    Those and their dependant targets will be generated as sources. \
+    If no target is specified, the project defined targets will be focused.
+    """)
+    var sources: [String] = []
+
+    @Flag(
+        name: .shortAndLong,
+        help: "Don't open the project after generating it."
+    )
+    var noOpen: Bool = false
+
     @Flag(
+        name: [.customShort("x"), .long],
```

Comment:
Personally not a fan of the short flags especially here because -x may read as "exclude". 

Id just keep the long name for the flag

Reply:
I agree, I was just keeping the old name